### PR TITLE
Modify sonar qube

### PR DESCRIPTION
**Pull request by ahsan-storm**
#### What this PR does / why we need it:
* Adds support for additional labels in sonarqube service

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: sonarqube
 description: Sonarqube is an open sourced code quality scanning tool
-version: 0.10.3
+version: 0.10.4
```
Comment (rjkernick) [inline stable/sonarqube/Chart.yaml]: Please bump the minor version instead since this adds a feature

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: stable/sonarqube/Chart.yaml (hunk 1 above)
Comment (ahsan-storm) [reply stable/sonarqube/Chart.yaml]: done, changed to 0.11.0

**Comment by ahsan-storm** [comment]
@unguiculus please merge it in master

**Pull request merged**